Core/VariantManager.swift

```diff
@@ -28,18 +28,21 @@ public enum FeatureName {
 public struct Variant {
 
     public static let defaultVariants: [Variant] = [
-        // None
+        
+        // For the SERP - do not remove:
+        Variant(name: "sc", weight: 1.0, features: []),
```

Comment:
How would you feel about using ints for the weights? It feels like overkill to allow both arbitrary values and allow them to be doubles.